SPEC.md

```diff
@@ -26,7 +26,19 @@ Any feedback or questions on this document should be directed to specifications
 THESE MATERIALS ARE PROVIDED “AS IS.” The Contributors and Licensees expressly disclaim any warranties (express, implied, or otherwise), including implied warranties of merchantability, non-infringement, fitness for a particular purpose, or title, related to the materials.  The entire risk as to implementing or otherwise using the materials is assumed by the implementer and user. IN NO EVENT WILL THE CONTRIBUTORS OR LICENSEES BE LIABLE TO ANY OTHER PARTY FOR LOST PROFITS OR ANY FORM OF INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES OF ANY CHARACTER FROM ANY CAUSES OF ACTION OF ANY KIND WITH RESPECT TO THIS DELIVERABLE OR ITS GOVERNING AGREEMENT, WHETHER BASED ON BREACH OF CONTRACT, TORT (INCLUDING NEGLIGENCE), OR OTHERWISE, AND WHETHER OR NOT THE OTHER MEMBER HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ## Introduction
-Type text.
+
+Modern Software Supply Chains face many challenges: security, compliance, provenance, and others.  All of them are hamstrung by the inability to easily and reliably capture and express a complete, concise, verifiable accounting of exactly **what** inputs where built into software.
+
+OmniBOR applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of "do one thing, and do it well." to construct a complete, concise, and verifiable Artifact Dependency Graph (ADG) for every software artifact enabling:
```

Comment:
How important do we think bringing up the Unix philosophy is? I know it's been part of the text of the website homepage for a while, but I wonder if mentioning it, especially before we've made more concrete arguments about the benefits of OmniBOR, is premature.

Reply:
It was intended as a table setting piece.  When someone asks 'Why doesn't OmniBOR do x?', Unix Philosophy is the answer.